Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/view/ListPlannerFragment.java

```diff
@@ -0,0 +1,324 @@
+    }
+
+    public void onResume() {
+        super.onResume();
+        getListView().invalidateViews();
+    }
+
+    @Override
+    public void onDateChanged(String date) {
+        //Not Used
+    }
+
+    @Override
+    public void onJumpTo(String date) {
+        //Not Used
+    }
+
+    public interface Callbacks {
+        void onListUpdate(String date);
+
+        void onTimeSlotSelected(String time, String date, String name, String place, String descr, String priority, String task);
+    }
+
+
+    private class ListFragmentAdapter extends ArrayAdapter<String> {
+
+        public ListFragmentAdapter(String[] time_slot) {
+            super(getActivity(), android.R.layout.simple_list_item_1, time_slot);
+        }
+
+        @Override
+        public View getView(int position, View convertView, ViewGroup parent) {
+            if (convertView == null) {
+                convertView = getActivity().getLayoutInflater().inflate(R.layout.daily_list_item, null);
+            }
+
+            String time = getItem(position);
+
+            mTimeSlot = (TextView) convertView.findViewById(R.id.time_slot);
+            mTimeSlot.setText(time);
+
+            mTextTask = (TextView) convertView.findViewById(R.id.task_name);
+            mTextTask.setText(getString(R.string.default_task_text));
+
+            mDescription = (TextView) convertView.findViewById(R.id.task_description);
+            mDescription.setText("No Description");
+
+            mLayout = (LinearLayout) convertView.findViewById(R.id.list_item_back);
+            mLayout.setBackgroundColor(Color.WHITE);
+
+            mListTasks = mTaskManager.getTasksForDate(getActivity().getTitle().toString());
+
+            Log.d("Planner", getActivity().getTitle().toString());
+            try {
+                mTextTask.setText(getString(R.string.default_task_text));
+                if (mListTasks == null) {
+                    return convertView;
+                }
+                for (Tasks t : mListTasks) {
+                    if (t.getDate().equals(getActivity().getTitle().toString())) {
+                        if (t.getTimeSlot().equals(mTimeSlot.getText().toString())) {
+                            mTextTask.setText(t.getTaskName());
+
+                            mDescription.setText(t.getDescription());
+                            if (t.getPriority().equals("HIGH")) {
+                                mLayout.setBackgroundColor(Color.RED);
+                                mTextTask.setTextColor(Color.DKGRAY);
+                                mDescription.setTextColor(Color.DKGRAY);
+                            } else {
+                                mLayout.setBackgroundColor(Color.GREEN);
+                                mTextTask.setTextColor(Color.DKGRAY);
+                                mDescription.setTextColor(Color.DKGRAY);
+                            }
+                            break;
```

Comment:
What is this "break" for?
Also Lines 296~309 (aside from the previous comment), I think can be moved to a different function instead.
Just call that function inside here to simplify the codes. :dart: